README.md

```diff
@@ -11,15 +10,20 @@ Install cookiecutter.
 pip install cookiecutter
 ```
 
-Use it to generate a package, following prompts fill in the name and authorship
-of your new JupyterLab extension.
+Use it to generate a package, following the prompts to fill in the name and authorship of your new JupyterLab extension.
 
 ```
 cookiecutter https://github.com/jupyter/jupyterlab-extension-cookiecutter
 ```
 
-A very simple working extension, written in common JavaScript, is included in
-the ``lib/`` directory. Use this example to build your own extension.
+A very simple working extension, written in Javascript using CommonJS modules, is included in the ``lib/`` directory. Use this example to build your own extension.
 
-User and developer installation instructions are included in the README of the
-package generated by cookiecutter.
+User and developer installation instructions are included in the README of the package generated by cookiecutter.
+
+## Package names
+
+JupyterLab extensions generated by this cookie cutter recipe are composed of both a python package and an npm package. The npm package is declared as a private package, so it cannot be published to the public npm package repository. The npm package is used to retrieve npm dependencies and provide a framework for the bundling of the javascript. The python package is published to the PyPI python package repository, and contains the generated JupyterLab extension Javascript bundle.
```

Comment:
This cookiecutter template creates a JupyterLab extension composed of a Python package and an npm package:

- The *npm package* is declared as a private package, so it cannot be published to the public npm package repository. The npm package is used to retrieve npm dependencies and provide a framework for the bundling of the JavaScript. 

- The *Python package* is published to PyPI, the Python package repository, and contains the generated JupyterLab extension JavaScript bundle.